src/openms/include/OpenMS/METADATA/SpectrumMetaDataLookup.h

```diff
@@ -225,25 +223,26 @@ namespace OpenMS
 
     /**
      * @brief encapsulates readSpectra with file loading convenience if only a file path is given
+     *
      * @param spectra_file file to the SpectrumContainer, must be valid path to mz file
      */
-    void readMzFile(const String& spectra_file)
+    void readMzFileMetaData(const String& spectra_file)
```

Comment:
I thought that maybe spectrumreference annotation is not the only usecase where you might want to have the SpectrumLookup object already setup with the metadata and the spectrum reference prior...